usaspending_api/disaster/v2/views/spending_by_geography.py

```diff
@@ -141,12 +144,22 @@ def build_elasticsearch_search_with_aggregation(self, filter_query: ES_Q) -> Opt
             bucket_count += 1
 
         # Add 100 to make sure that we consider enough records in each shard for accurate results
+        # We have to group by state no matter what since congressional districts and counties aren't unique between states
```

Comment:
When I first saw this I was hoping to find a more dynamic way of doing this to reduce the repeated code but when I started working on it, I kept noticing the subtle differences between them and realized how much of an undertaking that is. Definitely not blocking, just thinking we might need to look it over again at some point down the road.